framework/src/test/java/tests/ReturnsReceiverLombokTest.java

```diff
@@ -0,0 +1,37 @@
+package tests;
+
+import java.io.File;
+import java.util.List;
+import org.checkerframework.common.returnsreceiver.ReturnsReceiverChecker;
+import org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * tests the returns receiver checker's lombok integration, please not that the test files have been
```

Comment:
not -> note

Also, I think this comment needs to be updated given the new strategy for delombok'ing the tests.